### ISSUE TITLE
unexpected result

### ISSUE THREAD

**Issue by natxopedreira** [ppogg/YOLOv5-Lite#92]
Hello

Sorry if this is more a question that an issue, i'm not sure if i'm doing something wrong.

I trained a new model, using the v5lite-s weight  with "person" class using coco + voc dataset, the 1k background images as negative samples and to have an input of 320. I did not change any hyperparameter and using ncnn framework

I get:
mAP 05:0.95 of 0.498
mAP 05 of 0.795
Precision 0.84
Recall 0.68

I was expecting to have a better result than the provided model trained on all coco classes... but as you can see there is at least two persons not detected.

Maybe is a problem on the model conversion?

Thanks

**Standard coco model**
<img src="https://i.ibb.co/wYS3jrQ/result-coco.jpg" alt="result-coco" border="0">

**New coco_voc model**
<img src="https://i.ibb.co/JdxXN5f/result-coco-voc.jpg" alt="result-coco-voc" border="0">

**Comment by ppogg** [comment]
Hi, You are welcome to use YOLOv5-Lite. And I don’t understand what you mean, is your coco+voc total of 70,000 pieces of data?Or just use 1000 voc data for finetune on coco's pretraining model?

**Comment by natxopedreira** [comment]
Hi, yep my dataset is the full coco annotations and full voc for class "person" so yes arround 70000 images.

I mean i included 1000 "non person" images as negative samples, in the coco dataset there is around 1k of images without annotations that contais no person

**Comment by ppogg** [comment]
Generally, the detection frame with full screen is caused by the following reasons：
[https://github.com/ppogg/YOLOv5-Lite/issues/89](https://github.com/ppogg/YOLOv5-Lite/issues/89)
In addition, I noticed that your recall is very low. You can try lowering the threshold, or use 640*640 for training and 352*352 for inference.

**Comment by natxopedreira** [comment]
Thank you, i will train in 640x480.

I dont understand whats wrong with the param file as you pointed in #89 can you please explain? i think part of the info is lost when translating the comments.

Thanks a lot

My param last lines are
```
`Permute                  Transpose_469            1 1 650 output 0=1
Convolution              Conv_470                 1 1 611_splitncnn_0 652 0=18 1=1 5=1 6=2304
Reshape                  Reshape_484              1 1 652 670 0=400 1=6 2=3
Permute                  Transpose_485            1 1 670 671 0=1
Convolution              Conv_486                 1 1 631 672 0=18 1=1 5=1 6=4608
Reshape                  Reshape_500              1 1 672 690 0=100 1=6 2=3
Permute                  Transpose_501            1 1 690 691 0=1`
```

an opening in netron the onnx i get as output

<a href="https://ibb.co/9hnRpF7"><img src="https://i.ibb.co/2SqXFDT/Screenshot-2021-12-28-10-38-49.png" alt="Screenshot-2021-12-28-10-38-49" border="0"></a>

**Comment by ppogg** [comment]
Hi, This is a tutorial：
[https://blog.csdn.net/weixin_45829462/article/details/119787840](https://blog.csdn.net/weixin_45829462/article/details/119787840)

**Comment by natxopedreira** [comment]
Thanks!

**Comment by natxopedreira** [comment]
> Hi, This is a tutorial： https://blog.csdn.net/weixin_45829462/article/details/119787840
@ppogg 

Is hard to follow that one because website ask for login to complete read and google translate do not work on all the website so im unable to register to read that as i dont read chinesse.

Can this be seen somewhere that do not ask for a registration? Maybe you can export webpage and post link somewhere?

Thanks!
